kernel/src/thread/oops.rs

```diff
@@ -82,15 +77,20 @@ static OOPS_COUNT: AtomicUsize = AtomicUsize::new(0);
 #[ostd::panic_handler]
 fn panic_handler(info: &core::panic::PanicInfo) -> ! {
     let message = info.message();
+    let location = info.location();
 
     if let Some(thread) = Thread::current() {
         let panic_on_oops = PANIC_ON_OOPS.load(Ordering::Relaxed);
-        if !panic_on_oops && info.can_unwind() {
+
+        let should_handle = (!panic_on_oops && info.can_unwind())
+            || format!("{}", message).contains("not yet implemented")
```

Comment:
Use `PanicInfo::as_str` method instead to avoid heap allocation as much as possible in panic handling logic.